src/shared/components/SearchInput/SearchInput.jsx

```diff
@@ -55,27 +65,36 @@ function SearchInput({ items, setChosenItem, inputName, inputPlaceholder }) {
         type="text"
         className="search-input__input"
       />
-      <div ref={ref} className="search-input__list-items">
-        {isCountriesListExpanded ? (
-          <ul className="search-input__list">
-            {filteredItems.map(({ name, code }) => (
-              <li className="search-input__item" key={code}>
-                <Button
-                  className="search-input__item-button"
-                  type="button"
-                  onClick={() => onItemsSelect(name)}
-                  text={name}
-                />
-              </li>
-            ))}
-          </ul>
-        ) : null}
-      </div>
+      {isSearchingItems ? (
+        <Spinner />
+      ) : (
+        <div ref={ref} className="search-input__list-items">
+          {isCountriesListExpanded ? (
+            <ul className="search-input__list">
+              {isloading ? (
+                <Spinner />
+              ) : (
+                filteredItems.map(({ name, code }) => (
+                  <li className="search-input__item" key={code}>
+                    <Button
+                      className="search-input__item-button"
+                      type="button"
+                      onClick={() => onItemsSelect(name)}
+                      text={name}
+                    />
+                  </li>
+                ))
+              )}
+            </ul>
+          ) : null}
+        </div>
+      )}
```

Comment:
Nesting multiple ternary is not a goof practice. Do something like this before main` return`
```
if ({isSearchingItems) {
  return <Spinner />
}
```

Reply:
Fixed